docs/CONTRIBUTING.md

```diff
@@ -4,80 +4,83 @@ The BC Computer Science Club website is an open source project and everyone in t
 
 ## Table of Contents
 - [Getting Started](#getting-started)
-- [Branching Model](#branching-model)
 - [Using GitHub Issues](#using-github-issues)
 - [Making Changes](#making-changes)
 - [Submitting Pull Requests](#submitting-pull-requests)
-- [Review Process](#review-process)
+- [Code Review Process](#code-review-process)
 - [License](#license)
 
 ## Getting Started
 
-1. **Fork the repository**: Click the "Fork" button at the top right corner of the repository page.
-2. **Clone your fork**: Clone the forked repository to your local machine.
+1. **Fork the upstream repository**: Click the "Fork" button on https://github.com/bc-compsci-club/bccs-club
+2. **Clone your fork**: Clone the forked repository (NOT the upstream repository) to your local machine.
```

Comment:
I see so I need to fork the original repo and the clone my forked repo to VS Code.